graylog2-server/src/main/java/org/graylog2/inputs/codecs/JsonPathCodec.java

```diff
@@ -91,11 +114,17 @@ protected Map<String, Object> read(String json) {
         return fields;
     }
 
-    @VisibleForTesting
+    @VisibleForTesting //TODO change to no longer include jsonpath string, when full is selected
     protected String buildShortMessage(Map<String, Object> fields) {
         final StringBuilder shortMessage = new StringBuilder();
-        shortMessage.append("JSON API poll result: ");
-        shortMessage.append(jsonPath.getPath()).append(" -> ");
+        if (Objects.equals(modeString, "path")) {
```

Comment:
You can do this easier like this:

```suggestion
        if ("path".equals(modeString)) {
```